examples/rust/Cargo.toml

```diff
@@ -0,0 +1,8 @@
+[package]
+name = "olm-rustapp"
+version = "0.1.0"
```

Comment:
We'll want to have a single source of truth for version numbers, like maybe a top-level VERSION file that in turn is used to populate all the various files that need version numbers.